docs/meshstack.how-to.delete-meshObjects.md

```diff
@@ -0,0 +1,50 @@
+---
+id: meshstack.how-to.delete-meshObjects
+title: How to delete meshObjects in the meshPanel
+---
+
+Part of the cloud lifecyle inside meshStack is the deletion of the meshCustomers, meshProject and meshTenants.
+The following guide provides insight for users and admins how to proceed the deletion of their cloud accounts and meshObjects via the meshPanel
+
+## Step-by-Step Guide
+
+## Delete meshTenant from my meshProject
+
+If you want to delete just a meshTenant e.g. AWS Account without deleting your complete meshProject you can use the meshTenant deletion.
+
+The following step-by-step guide shows how to successfully request the deletion of an meshTenant:
+
+1. Go to your meshCustomer where the meshProject is located in the meshPanel ![Choose meshCustomer](./assets/customer/choose-customer.png)
+2. Go to your meshProject in which you want to delete the meshTenant ![Choose meshProject](./assets/project/project-in-top-nav.png)
+3. Click on "View more details" for the meshTenant you want to delete !["View more details"](./assets/project/project-nav-to-tenant.png)
+4. Click on the "Deletion" tab and follow the deletion instusction for the meshTenant !["Delete meshTenant"](./assets/project/project-tenant-deletion.png)
```

Comment:
please put the tenant deletion approval below it and make transparent that this needs to be done by an admin

Reply:
done